tests/app/questionnaire/test_placeholder_renderer.py

```diff
@@ -326,7 +326,7 @@ def get_placeholder_render_dynamic_answers(*, language="en", data_stores=DataSto
     )
 
 
-def get_placeholder_render(*, language="en", data_stores=DataStores()):
+def get_placeholder_render(*, language="en", data_stores=None):
     schema = MagicMock()
```

Comment:
if `data_stores` _can_ be None, would be good to cover for it, at the moment it could be called without being passed in, and then not work, with the other comment I'd imagined something like
```python
def x(data_stores=None):
    data_stores = data_stores or DataStores()
```
avoiding mutable default but allowing it not to be passed in, so all the calls to this don't need `get_placeholder_renderer(DataStores())`
atm we don't get the benefit from the default

Reply:
Done, thanks.